### ISSUE TITLE
Not removing HTML tags that aren't converted to markdown.

### ISSUE THREAD

**Issue by robert-groensfeld** [Elephant418/Markdownify#25]
Example:
```
<span><u>Some text</u></span>
```
This text will be the same after converting it to markdown, but the result should rather be `Some text`.

**Comment by tzi** [comment]
Hi!

Like you said, there is no equivalent to the `<u>` tag in Markdown, so we can not convert it.
Hopefully, HTML tags are valid Markdown, so the choice here is to keep the `<u>` to conserve the original formating.

There is an issue here, the `setKeepHTML()` method doesn't strip this tag :disappointed: 

I will look into it.
Thanks for your feedback.

Cheers,
Thomas.

**Comment by tzi** [comment]
It's fixed with the version v2.2.2 :tada: 

``` php
$converter = new Converter();
$converter->setKeepHTML(false);
echo $converter->parseString('<span><u>Some text</u></span>');
// Output "Some text"
```
